### ISSUE TITLE
怀疑这里有问题

### ISSUE THREAD

**Issue by little-polka-dot** [JiaquanYe/TableMASTER-mmocr#37]
master_decoder.py 271~272行

for layer in self.cls_layer:
    cls_x = layer(x, feature, src_mask, tgt_mask)

改成

cls_x = x
for layer in self.cls_layer:
    cls_x = layer(cls_x, feature, src_mask, tgt_mask)

其他地方也有类似问题

**Comment by JiaquanYe** [comment]
> 

Yes, you're right. It's doesn't go wrong because of 'self.cls_layer' length is 1.
Your solution is more reasonable.
